client/app/scripts/charts/nodes-chart.js

```diff
@@ -131,6 +131,13 @@ export default class NodesChart extends React.Component {
       return 1;
     };
 
+    const metric = node => {
+      const met = node.get('metrics') && node.get('metrics')
+        .filter(m => m.get('id') === this.props.selectedMetric)
+        .first();
+      return met;
+    };
+
```

Comment:
Pull this out and maybe pop the rest of the metrics off the nodes too, so a change in one of their values doesn't force a re-render of the chart even if its not selected.